### PR TITLE
php 8.2 support

### LINKED ISSUE #507: Can't install browscap-php with php 8.2
PHP 8.2.0 is out, but php version in composer.json is limited to 8.1.x

### PR DESCRIPTION
**Pull request by thirsch**
local tests are running fine and ignoring the dependency, it's running on a project fine as well. Fixes #507

### CODE REVIEW

**Review by mimmi20 — APPROVED** [review]

**Comment by asgrim** [comment]
@mimmi20 Hmm, we have failing tests here; 

 - https://github.com/browscap/browscap-php/actions/runs/3800232313/jobs/6463689481
 - https://github.com/browscap/browscap-php/actions/runs/3800232313/jobs/6463691266

File: composer.lock [hunk 1]
```diff
@@ -335,43 +335,48 @@
...
-                "url": "https://api.github.com/repos/thephpleague/flysystem/zipball/9392c5f1df57d865c406ee65e5012d566686be12",
-                "reference": "9392c5f1df57d865c406ee65e5012d566686be12",
+                "url": "https://api.github.com/repos/thephpleague/flysystem/zipball/2aef65a47e44f2d6f9938f720f6dd697e7ba7b76",
+                "reference": "2aef65a47e44f2d6f9938f720f6dd697e7ba7b76",
                 "shasum": ""
             },
             "require": {
-                "ext-json": "*",
                 "league/mime-type-detection": "^1.0.0",
-                "php": "^7.2 || ^8.0"
```
Comment (mimmi20) [inline composer.lock]: Please revert

**Review by mimmi20 — CHANGES_REQUESTED** [review]
Please rebase the PR to the current HEAD and then run `composer update --prefer-lowest`. This may get the failing test run.

File: composer.lock (hunk 1 above)
Comment (thirsch) [reply composer.lock]: Done, could you please restart the tests?

**Review by mimmi20 — APPROVED** [review]

**Comment by mimmi20** [comment]
> @mimmi20 Hmm, we have failing tests here; ...

@asgrim this is solved, please review again

**Review by asgrim — APPROVED** [review]
LGTM :shipit:

**Comment by asgrim** [comment]
@thirsch and @mimmi20 thank you both! [`browscap/browscap-php:7.2.0`](https://github.com/browscap/browscap-php/releases/tag/7.2.0) is now released with PHP 8.2 support :tada:

**Pull request merged**